.github/workflows/maven-deploy.yml

```
@@ -17,6 +17,26 @@ jobs:
     - name: Build with Maven
       run: mvn -B package
 
+    - name: Generate sbom output
+      run: mvn org.cyclonedx:cyclonedx-maven-plugin:makeBom
```

To put the json in the expected spot
```suggestion
      run: mvn org.cyclonedx:cyclonedx-maven-plugin:makeBom && mv ./duo-client/target/cyclonedx-sbom.json ./cyclonedx-sbom.json
```